components/admin_console/team_channel_settings/channel/channels.jsx

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
This file, again, unexpected about the filename, and the relation with the component file, I would expect a directory  `channel_list` with `index.jsx` inside with this data, and the `channel_list.jsx` inside that directory. Again, inconsistent with the existing approach.